templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,13 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 39**
+
+* Include `orig_subject` in update message events where the stream is
+changed, in addition to the existing inclusion when the topic is changed.
+NOTE: This change happened between feature levels 9 and 10, but due to
+an error, did not receive its own feature level at the time.
```

Comment:
This text should go in the "Feature level 9/10" section if we're just correcting documentation here, and the server's behavior was as described in this new version at that time.

The purpose of this changelog is to allow one to determine how the server of a given version functioned, not to document the sequences of changes in the documentation.

Reply:
I was following the comment [here](https://github.com/zulip/zulip/blob/58a44698963ffcd21b57937a4913562a032c5631/version.py#L23-L30) above `API_FEATURE_LEVEL` in `version.py`, which says:

```
# Bump the API_FEATURE_LEVEL whenever an API change is made
# that clients might want to condition on.  If we forget at
# the time we make the change, then bump it later as soon
# as we notice; clients using API_FEATURE_LEVEL will just not
# use the new feature/API until the bump.
#
# Changes should be accompanied by documentation explaining what the
# new level means in templates/zerver/api/changelog.md.
```

Did I misunderstand, or does that need to be changed?